server/meshmodel/schemas/relationship.json

```diff
@@ -79,8 +94,13 @@
             },
             "description": "Optional fields that are a part of the `to` selector. Absence of a field has an implied * meaning."
           }
+        },
+        "allowMultiple": {
```

Comment:
This helps us in knowing whether between the same two component multiple connection of same type is available,

Reply:
After reading the description it seems like there will 2 edges, pod to pvc and then pvc to pv.
Or this schema update is just facilitating the current behaviour we have

Reply:
Although we want to have some modification in the code written here. But to your question, it just tells about "via" node, and its on UI implementation to chose any of the two approaches you're mentioning.